roles/parse_operator_metadata/tasks/main.yml

```diff
@@ -118,39 +118,42 @@
       crd_paths: "{{ crd_paths + [item.path] }}"
     with_items: "{{ crd_paths_result['files'] }}"
 
-  - name: "Include the package path into the ansible vars"
-    include_vars:
-      file: "{{ csv_path }}"
-      name: operator_vars
+  - name: "Read csv.yaml file"
+    shell: "cat {{ csv_path }}"
+    register: csv_data
+
+  - name: "Set facts for csv_data"
+    set_fact:
+      csv_vars: "{{ csv_data.stdout }}"
```

Comment:
Shouldn't this be converted here and use csv_vars at all other recursive places ?

Reply:
Same case as mentioned above.

Reply:
from_yaml is not needed here?

Reply:
It is, updated.